west.yml

```diff
@@ -98,7 +98,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 7276046215cfcd4ed3d6723f16f7aeb868b0fb60
+      revision: pull/51/head
```

Comment:
The nxp_hal PR has now been merged, so you can update this to the new SHA

Reply:
ok， i will update this